packages/a11y/src/index.ts

```diff
@@ -0,0 +1,75 @@
+/* tslint:disable:no-console */
+import path from 'path';
+import puppeteer from 'puppeteer';
+import {WebpackConfigurator, serve, IServer, waitForPageError} from 'ui-autotools-utils';
+import axe from 'axe-core';
+import chalk from 'chalk';
+
+const ownPath = path.resolve(__dirname, '..');
+const impactArray: axe.ImpactValue[] = ['minor', 'moderate', 'serious', 'critical'];
+
+function getWebpackConfig(p: string) {
+  const webpackConfigPath = path.join(process.cwd(), p, 'meta.webpack.config.js');
+  return WebpackConfigurator
+    .load(webpackConfigPath)
+    .addEntry('meta', path.join(ownPath, 'esm/browser/run'))
+    .addHtml({
+      template: path.join(ownPath, 'src/browser/index.html'),
+      title: 'Accessibility'
+    })
+    .suppressReactDevtoolsSuggestion()
+    .getConfig();
+}
+
+function printResults(results: axe.AxeResults, impact: number): string {
+  const msg: string[] = [];
+  let index = 1;
+  results.violations.forEach((violation) => {
```

Comment:
A second argument to forEach gives you the index, so it's not really necessary to define the index variable and increment it manually.